packages/zowe-explorer/__tests__/__unit__/abstract/ZoweSaveQueue.unit.test.ts

```diff
@@ -14,12 +14,20 @@ import { createUSSTree } from "../../../__mocks__/mockCreators/uss";
 import { saveUSSFile } from "../../../src/uss/actions";
 import * as workspaceUtils from "../../../src/utils/workspace";
 import { Gui } from "@zowe/zowe-explorer-api";
-import * as globals from "../../../src/globals";
 import * as vscode from "vscode";
 import { ZoweLogger } from "../../../src/utils/LoggerUtils";
+import { ZoweLocalStorage } from "../../../src/utils/ZoweLocalStorage";
 
 describe("ZoweSaveQueue - unit tests", () => {
     const createGlobalMocks = () => {
+        Object.defineProperty(ZoweLocalStorage, "storage", {
+            value: {
+                get: () => ({ persistence: true, favorites: [], history: [], sessions: ["zosmf"], searchHistory: [], fileHistory: [] }),
+                update: jest.fn(),
+                keys: () => [],
+            },
+            configurable: true,
+        });
```

Comment:
I've noticed this logic across a few of the test files. What are your thoughts on pulling this logic into a single "mock initialization function" that we can reference across all of these test files? That way, if anything needs changed with this object, it can be done in one place and that change will apply to all tests that use it.

Reply:
This is a good suggestion! I think if we want to go ahead and make this change it would be in a separate PR where we also address other redundant functions like the mocks for the logger right above these same exact mocks (perhaps we could add a globalMock that can just be imported into our test files instead of writing them everytime).

Reply:
Agreed, I think this can be pulled into its own PR alongside some of the other duplicate mocks in our codebase.